src/RefactoringGuru/Proxy/Structural/ProxyStructural.php

```diff
@@ -20,10 +20,10 @@ public function request();
 }
 
 /**
- * The Real Subject contains some core business logic. Usually, RealSubjects are
- * able of doing some useful work, which is also very slow or sensitive to
- * correct input data. A Proxy can solve these issues without any changes to the
- * real subject's code.
+ * The RealSubject contains some core business logic. Usually, RealSubjects are
+ * capable of doing some useful work which may also be very slow or sensitive -
+ * e.g. correcting input data. A Proxy can solve these issues without any 
+ * changes to the real subject's code.
```

Comment:
How about this:

to the real subject's code → to the RealSubject's code.

Reply:
Good catch, that is more consistent. I've pushed a new commit with the change.